.codeclimate.yml

```diff
@@ -42,3 +42,4 @@ exclude_patterns:
   - "static/"
   - "t/"
   - "images/qr-assets.go"
+  - "contracts/hop/l2Contracts/l2BaseBridge/l2BaseBridge.go"
```

Comment:
hmm @igor-sirotin it seems there's some auto-generated code that's not getting auto-excluded, is there some other place where this needs to be added?